modules/llm-cache/radix-tree/radix-tree.cc

```diff
@@ -60,30 +60,30 @@ RadixTree::~RadixTree() {
 }
 
 std::shared_ptr<NodeData> RadixTree::Insert(
-    std::vector<int> tokens, std::shared_ptr<NodeData>& evictedNode) {
+    std::vector<int>& tokens, std::shared_ptr<NodeData>& evictedNode) {
   tokens.insert(tokens.begin(), INT32_MAX);
   return InsertInternal(tokens, evictedNode);
 }
 
-void RadixTree::Delete(std::vector<int> tokens,
+void RadixTree::Delete(std::vector<int>& tokens,
                        std::shared_ptr<NodeData>& evictedNode) {
   tokens.insert(tokens.begin(), INT32_MAX);
   DeleteInternal(tokens, evictedNode);
 }
 
-std::shared_ptr<NodeData> RadixTree::Query(std::vector<int> key) {
+std::shared_ptr<NodeData> RadixTree::Query(std::vector<int>& key) {
   key.insert(key.begin(), INT32_MAX);
   return QueryInternal(key);
 }
 
 std::vector<std::shared_ptr<NodeData>> RadixTree::Split(
-    std::vector<int> tokens, std::shared_ptr<NodeData>& header) {
+    std::vector<int>& tokens, std::shared_ptr<NodeData>& header) {
   tokens.insert(tokens.begin(), INT32_MAX);
   return SplitInternal(tokens, header);
 }
 
 std::shared_ptr<NodeData> RadixTree::InsertInternal(
-    std::vector<int> tokens, std::shared_ptr<NodeData>& evictedNode) {
+    std::vector<int>& tokens, std::shared_ptr<NodeData>& evictedNode) {
```

Comment:
Should be `const std::vector<int>& tokens` as it is the argument, not for return value.